### DISCUSSION TITLE
Will Silk.NET consider binding physics engines in the future?

### DISCUSSION THREAD

**Discussion by qian-o** [dotnet/Silk.NET#1673] [Ideas] [answered]
For the .NET ecosystem, there are currently very few options for physics engines, and many binding libraries lack support for non-Windows platforms, which makes the selection process quite frustrating.

**Comment by Beyley** [comment, marked as answer]
We generally try to bind to industry standard cross platform APIs, and tmk there isnt *really* a single "standard" physics library with a C API, although bindings to one of the many that do exist may be a good fit for SilkCommunity, and we would love to onboard a set of physics engine bindings using our generator!
